src/logic/operate.js

```diff
@@ -20,5 +20,15 @@ export default function operate(numberOne, numberTwo, operation) {
       return one.div(two).toString();
     }
   }
+  if (operation === "**2") {
+    if(one === 0){
+      if(two === 1){
+        return 1
+      }
+      else return 0
+    }
+    else
+      return one.times(one).toString();
```

Comment:
Codacy found an issue: [Expected no linebreak before this statement.](https://app.codacy.com/gh/dan-codacy/calculator/pullRequest?prid=6272202)